flask_security/forms.py

```diff
@@ -195,7 +195,7 @@ def __init__(self, *args, **kwargs):
     def validate(self):
         if not super(PasswordlessLoginForm, self).validate():
             return False
-        if not self.user.is_active():
+        if not self.user.is_active:
```

Comment:
in #417 this is not changed but test are passing is it necesary to also change this with Flask-Login 0.3.0? maybe these lines are not covered by any test. i can change the PR if required